setup.cfg

```
@@ -26,16 +26,18 @@ classifiers =
   License :: OSI Approved :: BSD License
   Natural Language :: English
   Programming Language :: Python :: 3.6
-  Programming Language :: Python :: 3.7
   Programming Language :: Python :: 3.8
+  Programming Language :: Python :: 3.9
+  Programming Language :: Python :: 3.10
   Operating System :: MacOS :: MacOS X
   Operating System :: POSIX :: Linux
+  Operating System :: Microsoft :: Windows
 license_file = LICENSE
 long_description = file: README.md
 long_description_content_type = text/markdown
 
 [options]
-python_requires = >= 3.5
+python_requires = >= 3.6.8
```

```suggestion
python_requires = >= 3.6
```

I recommend making this simple, even though we test the particular version I needed to work.